### ISSUE TITLE
Make sure there are no distance to transit with values = 0 in buffered_parcels

### ISSUE THREAD

**Issue by stefancoe** [psrc/soundcast#276]
Some parcels will share the same network node as a transit stop and have a distance of 0. I think Daysim may consider this not accessible. Add code to accessibility.py to re-code these to .01.
